skrifa/src/patchmap.rs

```diff
@@ -3,18 +3,19 @@
 //! The IFT and IFTX tables encode mappings from subset definitions to URL's which host patches
 //! that can be applied to the font to add support for the corresponding subset definition.
 
+use core::ops::RangeInclusive;
```

Comment:
this should be `std` (which is aliased to `core` if real 'std' is unavailable) but rust-analyzer will sometimes use `core` instead when auto-importing in a crate that has no-std support...